app/routes/studio_assets.py

```
@@ -636,6 +741,141 @@ async def generate():
+                .execute()
+            )
+            if result.data:
+                session_id = result.data[0]["id"]
+        except Exception:
+            logger.exception("Failed to find test session for memory save")
+
+        # Fire-and-forget: save memory before clearing
+        if session_id:
+            asyncio.create_task(
+                cko_service.save_test_session_memory(session_id, user_id, asset_id)
+            )
```

Ruff (0.15.2)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)